securedrop_client/gui/widgets.py

```diff
@@ -91,25 +145,31 @@ def setup(self, window, controller):
         self.window = window
         self.controller = controller
 
-        self.controller.sync_events.connect(self._on_sync_event)
-
     def set_logged_in_as(self, username):
         """
         Update the UI to reflect that the user is logged in as "username".
         """
-        self.user_state.setText(html.escape(username))
-        self.login.setVisible(False)
-        self.logout.setVisible(True)
-        self.refresh.setVisible(True)
+        self.login.hide()
+
+        self.user_icon.setText(_('jo'))
+        self.user_icon.setStyleSheet('''
+            QLabel { background-color: #045fb4; color: cyan; padding: 10; border: 1px solid gray; }
+        ''')
+        self.user_icon.show()
+
+        self.user_state.setText(_('{}').format(html.escape(username)))
+        self.user_state.show()
+
+        self.user_menu.show()
 
     def set_logged_out(self):
         """
         Update the UI to a logged out state.
         """
-        self.user_state.setText(_('Signed out.'))
-        self.login.setVisible(True)
-        self.logout.setVisible(False)
-        self.refresh.setVisible(False)
```

Comment:
I think when we log out we want the refresh button to be hidden no?

Reply:
@creviera asked me about where the Offline icon was, yesterday—did you want to put that in on a separate PR or were you waiting for me to respond to your Slack, Allie? All icons are in table on this Issue: https://github.com/freedomofpress/securedrop-ux/issues/17

Reply:
Oh yeah that's right! The new offline icon will be in the pr for this issue: https://github.com/freedomofpress/securedrop-client/issues/279. As I understand it, there will be three icons which are in this inventory list: https://github.com/freedomofpress/securedrop-ux/issues/17. There you'll see an icon labeled "offline", the static refresh icon, and the active refresh icon.

Realizing now that when I moved the refresh icon to the StatusBar class, I didn't add a test to make sure it's hidden, so I didn't catch that I changed the behavior. Good catch. However, I'm working on a follow-up PR to change and test the new behavior of refresh and, so if you're cool with leaving this as is for now, I'm cool wit it. :sunglasses:

Reply:
We discussed this and decided that since the refresh icon appearing when not logged in is a regression it would be good to resolve as part of this PR